### DISCUSSION TITLE
RFC: VersaTiles Frontend Specification

### DISCUSSION THREAD

**Discussion by MichaelKreil** [versatiles-org/versatiles-documentation#31] [Ideas]
Since different map servers organise the files for the frontend in different ways, I propose to define a standard way to organise map frontends: [Frontend Specification](https://github.com/versatiles-org/versatiles-documentation/blob/improve-frontend-specification/compendium/specification_frontend.md)

I appreciate any kind of feedback, since this will be the frontend file structure we'll use in all VersaTiles servers and frontends.

**Comment by MichaelKreil** [comment]
OK, merged ✅
New Link: [Frontend Specification](https://github.com/versatiles-org/versatiles-documentation/blob/main/compendium/specification_frontend.md)
Now I can start implementing

**Comment by yetzt** [comment]
maputnik expects a .json file to look up the available fonts, to my understanding it's inferred from the glyphs url. eg `https://maps.example.org/fonts/{fontstack}/{range}.pbf` → `https://maps.example.org/fonts.json`

we should not break that.
